### PR TITLE
Created a view component for the forgot Password form

### PR DESCRIPTION
**Pull request by OladipoOmotosho**
Created a view component for the forgot Password form

### CODE REVIEW

File: src/components/ForgotPasswordForm_comp.jsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import reset from "../assets/Icons/Light Mode Icon/reset.png";
+import Image from "next/image";
+const ForgotPassword_comp = () => {
```
Comment (annonymousauthority) [inline src/components/ForgotPasswordForm_comp.jsx:4]: No need for `_comp`
`ForgotPassword` is enough

File: src/components/ForgotPasswordForm_comp.jsx
Comment (annonymousauthority) [inline src/components/ForgotPasswordForm_comp.jsx:1]: Please could you be consistent with the extension?
`.jsx` --> `.js`

File: src/components/ForgotPassword.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import reset from "../assets/Icons/Light Mode Icon/reset.png";
+import Image from "next/image";
+const ForgotPassword = () => {
+  return (
+    <div className="w-full flex flex-col justify-center items-start bg-white rounded-3xl shadow-lg">
+      <span>
+        <h3 className="text-blue-500 sm:text-lg font-bold ">Forgot Password</h3>
+        <Image src={reset} alt="reset" />
```
Comment (annonymousauthority) [inline src/components/ForgotPassword.js:9]: This should betton not an image. 

the button would wrap an image

File: src/components/ForgotPassword.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import reset from "../assets/Icons/Light Mode Icon/reset.png";
+import Image from "next/image";
+const ForgotPassword = () => {
+  return (
+    <div className="w-full flex flex-col justify-center items-start bg-white rounded-3xl shadow-lg">
+      <span>
```
Comment (annonymousauthority) [inline src/components/ForgotPassword.js:7]: Please utilize a div here and use a `flex` and `justify between` and `w-full` here

File: src/components/ForgotPassword.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+        <h3 className="text-blue-500 sm:text-lg font-bold ">Forgot Password</h3>
+        <Image src={reset} alt="reset" />
+      </span>
+      <p className="text-base leading-5 text-blue-950 font-normal">
+        Enter your email address to reset your password
+      </p>
+      <form>
+        <label
```
Comment (annonymousauthority) [inline src/components/ForgotPassword.js:15]: You label should point to a htmlFor

File: src/components/ForgotPassword.js [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+          className="text-base leading-5 text-blue-950 font-normal"
+          htmlFor=""
+        >
+          Email Address
+        </label>
+        <input
```
Comment (annonymousauthority) [inline src/components/ForgotPassword.js:21]: You input should have a name and id.
And the `type` should be an `email` not a `text`

**Review by annonymousauthority — CHANGES_REQUESTED** [review]

**Comment by OladipoOmotosho** [comment]
Screenshot (144)
Screenshot (143)

**Comment by OladipoOmotosho** [comment]
Screenshots attached above

**Comment by OladipoOmotosho** [comment]
Screenshot (148)
Screenshot (150)

**Comment by OladipoOmotosho** [comment]
I added the black background for better visibility but I've removed it after taking the screenshot so it's not in the code I just pushed

**Review by annonymousauthority — APPROVED** [review]
LG. Although I couldn't find the padding update

**Comment by OladipoOmotosho** [comment]
I added it, I also attached screenshots of how it looks now as a commentSent from my T-Mobile 4G LTE Device
-------- Original message --------From: Augustine Francis ***@***.***> Date: 5/21/23  10:19 PM  (GMT+01:00) To: annonymousauthority/P2PCoder ***@***.***> Cc: Oladipo Omotosho ***@***.***>, Author ***@***.***> Subject: Re: [annonymousauthority/P2PCoder] Created a view component for the forgot Password form (PR #60) 
@annonymousauthority approved this pull request.

LG. Although I couldn't find the padding update

**Pull request merged**